clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -48,7 +48,7 @@
  * Here is pseudo-code for a callback implementation for saving offsets:
  * <pre>
  * {@code
- *   public class SaveOffsetsOnRebalance implements ConsumerRebalanceCallback {
```

Comment:
I was thinking that Listener makes it clear that it will be invoked multiple times. In the other cases we are using callbacks, they are invoked just once. I don't have a strong preference, however.